core/src/server/rpc_module.rs

```diff
@@ -889,7 +1021,7 @@ impl SubscriptionSink {
 	///
 	/// ```no_run
 	///
-	/// use jsonrpsee_core::server::rpc_module::{RpcModule, SubscriptionResult};
+	/// use jsonrpsee_core::server::rpc_module::RpcModule;
 	/// use jsonrpsee_core::error::{Error, SubscriptionClosed};
 	/// use jsonrpsee_types::ErrorObjectOwned;
 	/// use anyhow::anyhow;
```

Comment:
Update this example with new pipe_from_stream .on_success/.on_failure api?

Reply:
Ah; PipeFromStreamResult doesn't appear to be used in these `pipe_from_stream` and `pipe_from_try_stream` yet. Should we be using them here too for consistency?

Reply:
yes :100: